packages/venia-ui/lib/components/ForgotPassword/ForgotPasswordForm/forgotPasswordForm.js

```diff
@@ -25,8 +26,7 @@ const ForgotPasswordForm = props => {
                 <TextInput
                     autoComplete="email"
                     field="email"
-                    validate={isRequired}
-                    validateOnBlur
```

Comment:
Before @dpatil-magento merges maybe @revanth0212 can explain why we had to remove this. I feel like the validation should happen on blur _and_ before submit, but it seems like it wasn't happening correctly.

Reply:
The issue with `validateOnBlur` is that it only validates on blur. So once you enter a new valid email address and click on the submit button, the validation is not done yet and the click won't be registered because clicking button === losing focus for the email field but the validation result will only propagate in the next render. That is the reason why @dpatil-magento  had to click twice.